Library/Homebrew/cask/cask_loader.rb

```diff
@@ -30,14 +30,16 @@ def initialize(content)
         @content = content.force_encoding("UTF-8")
       end
 
-      def load
+      def load(config:)
```

Comment:
This has broken the formulae.brew.sh generation: https://github.com/Homebrew/formulae.brew.sh/blob/9afbae0ea8e80749568d6415a0a320a0c840b0a4/script/generate-cask.rb#L15

Could you submit a PR to fix @reitermarkus? Thanks!